tests/functional/codegen/features/test_transient.py

```diff
@@ -460,6 +490,8 @@ def bar(i: uint256, a: address) -> uint256:
 
     value = 333
     assert c2.bar(value, c1.address) == value
+    env.clear_transient_storage()
```

Comment:
i'd like to add tests that do multiple message calls into the same contract without updating transient storage (which we can do now that we control transactionality, compared to eth-tester!), although it makes sense to snip scope here and punt to a future PR